src/transforms/stdnames.jl

```diff
@@ -0,0 +1,48 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    StdNames(S)
+
+StdNames transform standardizes the column names.
+"""
+struct StdNames <: Stateless
+  spec::Symbol
+end
+
+StdNames() = StdNames(:upper)
+
+isrevertible(::Type{StdNames}) = true
+
+function apply(transform::StdNames, table)  
+  oldnames = Tables.columnnames(table)
```

Comment:
```suggestion
  cols = Tables.columns(table)
  oldnames = Tables.columnnames(cols)
```
This is the correct way to use the `Tables.columnnames` function.

Reply:
Thanks!